src/smf/packet-smf.c

```diff
@@ -861,32 +877,36 @@ static guint get_smf_pdu_len(packet_info* inf, tvbuff_t *tvb, int offset, void *
 }
 
 /* Add a base-64 encoded string to the tree */
-static void smf_proto_add_base64_string(proto_tree *tree, int id, tvbuff_t *tvb,
+static void smf_proto_add_base64_string(proto_tree *tree, packet_info *pinfo, int id, tvbuff_t *tvb,
     int offset, int size)
 {
     char* str;
     gsize len = size;
    
     str = tvb_get_string_enc(NULL, tvb, offset, size, ENC_ASCII);
     if (len > 1) {
-        g_base64_decode_inplace(str, &len);
+        if (strlen(str) > 1) {
+            g_base64_decode_inplace(str, &len);
+        } else {
+            g_print("smf g_base64_decode_inplace failed at packet %d, strlen is < 1", pinfo->fd->num);
+        }
     }
     str[len] = '\0';
```

Comment:
This should be after g_base64_decode_inplace() which is where 'len' gets set.  
Previous this might have null termianted 'str' as returned by tvn_get_string_enc() when we were misusing len and size, but if there is any reason to suspect that tvn_get_string_enc() does NOT return a null terminated string we should not be passing it to strlen().

Reply:
Inside g_base64_decode_inplace(), there is a check for strlen() > 1.
In some pcap, we are failing that check.
I am making that check explicit and printing the offending packet frame.

I think in this case, the problem is that the first character is null (\0).